.gitignore

```diff
@@ -1,3 +1,13 @@
+# .gitignore
```

Comment:
I would probably just git ignore the whole data folder

Reply:
I've [ignored the data files](https://github.com/myHaiven/data-collection/pull/4/commits/8dfb730a65ecfb872fb5e86b185d9a4f658f8621#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R4) by replacing the lines with just `data/`.